### DISCUSSION TITLE
Open Minecraft server assembly

### DISCUSSION THREAD

**Discussion by arslee07** [fdl-mc/freedomland#1] [Q&A] [answered]
...it will include PaperMC, Spigot, Bukkit, Tuinity and Purpur configurations, list of plugins which we use on a server and its configs.
What do you think about that idea?

**Comment by arslee07** [comment]
also i hav bad english so im sussy chungus among us

**Comment by arslee07** [comment]
> ...it will include PaperMC, Spigot, Bukkit, Tuinity and Purpur configurations, list of plugins which we use on a server and its configs. What do you think about that idea?

...and old seasons maps maybe? =)

**Reply by voidlynx** [reply]
абоба чангус

i'm all up for this idea, upvoted + reddit gold + wholesome 100

**Comment by voidlynx** [comment]
ezezez готово

**Comment by arslee07** [comment, marked as answer]
ez

**Comment by arslee07** [comment]
а нахуя открывал то

**Comment by arslee07** [comment]
ладно
